test/models/event_log_test.rb

```diff
@@ -104,4 +104,12 @@ class EventLogTest < ActiveSupport::TestCase
     assert_equal log.entry, EventLog::PASSPHRASE_RESET_REQUEST
     assert_not_nil log.created_at
   end
+
+  test "can record an account invitation" do
+    user = create(:user)
+    admin = create(:admin_user)
+
+    event_log = EventLog.record_account_invitation(user, admin)
+    assert_equal EventLog::ACCOUNT_INVITED, event_log.entry
```

Comment:
Again it'd be nice to assert that the `initiator` is set properly with the 2nd argument to the helper.